src/styles/release-candidate/components/_c-button.scss

```diff
@@ -1,21 +1,23 @@
 .c-button {
```

Comment:
Removed a bit of the dry sass mixins and helpers until we build out the components more, it feels like overkill to have for just a few different variations.